backend/src/main/java/gov/cdc/usds/simplereport/service/ExportService.java

```diff
@@ -22,22 +23,23 @@
 public class ExportService {
 
     private final TestEventRepository _ts;
-    private OrganizationService _os;
 
-
-    public ExportService(TestEventRepository ts, OrganizationService os) {
+    public ExportService(TestEventRepository ts) {
         this._ts = ts;
-        this._os = os;
     }
 
-    public String CreateTestEventCSV() throws IOException {
-        List<TestEvent> events = _ts.findAllByOrganizationOrderByCreatedAtDesc(
-            _os.getCurrentOrganization()
-        );
+    public String CreateTestEventCSV(String createAtDateThreshold) throws IOException {
```

Comment:
method name (which I realize escaped prior review)—use standard camelCase conventions.

Reply:
Fixed, but this function was longer being used, so I the solution was to delete the file.

Reply:
🏴‍☠️ Dead code throws no exceptions. 🦜